src/platforms/react-native/troubleshooting.mdx

```diff
@@ -242,3 +242,15 @@ React Native 0.60.3 tooling was unexpectedly moving source maps files, which cau
 ## Missing Java stacktrace on the New Architecture
 
 If you're using the new architecture on React Native, you might be missing the Java stacktrace. Follow the [issue](https://github.com/facebook/react-native/issues/34923) for more information.
+
+## Failing Build Due to `sentry-cli`
+
+If you're building your app without access to [sentry.io](https://sentry.io) or some other unexpected error occurs, you can allow failure of the `sentry-cli` step by adding an environmental variable `SENTRY_ALLOW_FAILURE` equal to `true`. The CLI will still run, but won't cause the build to fail:
```

Comment:
H: I'd give an example, such as internet issues or sentry.io is down.
I'd also explain that if this fails when eg uploading source maps, this should be done later manually once the connection is back.
I'd also explain how to remove the env. var. once everything is back to normal.

Reply:
Okay, I've adjusted the text.